docs/docs/cmd/spo/listitem/listitem-roleinheritance-break.md

```diff
@@ -0,0 +1,54 @@
+# spo listitem roleinheritance break
+
+Break inheritance of list item. Keeping existing permissions is the default behavior.
+
+## Usage
+
+```sh
+m365 spo listitem roleinheritance break [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: URL of the site from which the item should be retrieved
+
+`-i, --id <id>`
```

Comment:
Since our last noun is `roleinheritance`, this should be `--listItemId` to clearly specify that we're expecting the ID of the list item rather than roleinheritance

Reply:
✅